tests/ComputeSharp.Dynamic.NuGet/ComputeSharp.Dynamic.NuGet.csproj

```diff
@@ -1,15 +1,21 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net472;netcoreapp3.1;net6.0</TargetFrameworks>
+    <TargetFrameworks>net472;netcoreapp3.1;net6.0;net7.0</TargetFrameworks>
     <Platforms>AnyCPU;x64;ARM64</Platforms>
     <RestoreSources>
       https://api.nuget.org/v3/index.json;
       ..\..\artifacts;
     </RestoreSources>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net7.0'">
+    <PublishAot>true</PublishAot>
+    <EnableTrimAnalyzer>true</EnableTrimAnalyzer>
```

Comment:
Isn't you want `EnableTrimAnalyzer` for everything?

Reply:
I think it's fine to leave this just for .NET 7 for now, since it's the only target doing trimming in this test 🙂